FirebaseCore/Tests/Unit/Swift/FirebaseAppTests.swift

```diff
@@ -77,120 +86,104 @@ class FirebaseAppTests: XCTestCase {
 
-    XCTAssertNoThrow(FirebaseApp.configure(name: testApp.testAppName1, options: options1))
+    XCTAssertNoThrow(FirebaseApp.configure(name: Constants.testAppName1, options: options1))
 
-    // TODO: Use XCTUnwrap after dropping support for Xcode 10
-    guard let app1 = FirebaseApp.app(name: testApp.testAppName1) else {
-      return XCTFail("Failed to unwrap app1")
-    }
-    XCTAssertEqual(app1.name, testApp.testAppName1)
-    XCTAssertEqual(app1.options.googleAppID, testOptions.googleAppID)
-    XCTAssertEqual(app1.options.gcmSenderID, testOptions.gcmSenderID)
-    XCTAssertEqual(app1.options.deepLinkURLScheme, testOptions.deepLinkURLScheme)
+    let app1 = try XCTUnwrap(FirebaseApp.app(name: Constants.testAppName1), "Failed to unwrap app1")
+    XCTAssertEqual(app1.name, Constants.testAppName1)
+    XCTAssertEqual(app1.options.googleAppID, Constants.Options.googleAppID)
+    XCTAssertEqual(app1.options.gcmSenderID, Constants.Options.gcmSenderID)
+    XCTAssertEqual(app1.options.deepLinkURLScheme, Constants.Options.deepLinkURLScheme)
     XCTAssertTrue(FirebaseApp.allApps?.count == 1)
 
     // Configure a different app with valid customized options.
-    let options2 = FirebaseOptions(googleAppID: testOptions.googleAppID,
-                                   gcmSenderID: testOptions.gcmSenderID)
-    options2.bundleID = testOptions.bundleID
-    options2.apiKey = testOptions.customizedAPIKey
+    let options2 = FirebaseOptions(googleAppID: Constants.Options.googleAppID,
+                                   gcmSenderID: Constants.Options.gcmSenderID)
+    options2.bundleID = Constants.Options.bundleID
+    options2.apiKey = Constants.Options.apiKey
 
-    let expectedUserInfo2 = expectedUserInfoForApp(named: testApp.testAppName2,
-                                                   isDefaultApp: false)
-    expectAppConfigurationNotification(with: expectedUserInfo2)
+    expectAppConfigurationNotification(appName: Constants.testAppName2, isDefaultApp: false)
 
     let configureApp2Attempt = {
       try ExceptionCatcher.catchException {
-        FirebaseApp.configure(name: self.testApp.testAppName2, options: options2)
+        FirebaseApp.configure(name: Constants.testAppName2, options: options2)
       }
     }
     XCTAssertNoThrow(try configureApp2Attempt())
 
-    // TODO: Use XCTUnwrap after dropping support for Xcode 10
-    guard let app2 = FirebaseApp.app(name: testApp.testAppName2) else {
-      return XCTFail("Failed to unwrap app")
-    }
-    XCTAssertEqual(app2.name, testApp.testAppName2)
-    XCTAssertEqual(app2.options.googleAppID, testOptions.googleAppID)
-    XCTAssertEqual(app2.options.gcmSenderID, testOptions.gcmSenderID)
-    XCTAssertEqual(app2.options.bundleID, testOptions.bundleID)
-    XCTAssertEqual(app2.options.apiKey, testOptions.customizedAPIKey)
+    let app2 = try XCTUnwrap(FirebaseApp.app(name: Constants.testAppName2), "Failed to unwrap app")
```

Comment:
Nit: `failed to unwrap app2` to match the `app1` above

Reply:
done